packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/add-edit/index.js

```diff
@@ -0,0 +1,40 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName =
+  'ovhManagerPciStoragesDatabaseReplicationsAddEditLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider
+      .state(
+        'pci.projects.project.storages.databases.dashboard.replications.add.**',
+        {
+          url: '/add',
+          lazyLoad: ($transition$) => {
+            const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+
+            return import('./add-edit.module').then((mod) =>
+              $ocLazyLoad.inject(mod.default || mod),
+            );
+          },
```

Comment:
**nitpick**
Following Sonarcloud feedback, can you please refactor this part to avoid duplicating it?